### DISCUSSION TITLE
Binaries for testing PR's

### DISCUSSION THREAD

**Discussion by GaryElshaw** [transmission/transmission#3937] [Ideas]
Something i've been thinking about for a long time but #3922 brought it to the fore, can we surface the effected platform binaries from Team City into the PR page?

Not necessarily core changes, but it would be great to have an easy way to test platform-specific PR's.

**Comment by davischw** [comment]
Have a look at the [FRRouting Project](https://frrouting.org/)'s CI system. Packages for every built platform available as Artifacts for every iteration of every PR and for every merge to master/stable branches.

As an example for a specific PR: https://ci1.netdef.org/browse/FRR-PULLREQ2-7969/artifact

If you have any build automation already in place, this should not be hard to implement.
